go/vm/evmc/evmc_steppable_interpreter.go

```diff
@@ -62,6 +62,12 @@ func (e *SteppableEvmcInterpreter) StepN(
 		return nil, err
 	}
 
+	evmcBlobHashes := []evmc.Hash{}
```

Comment:
In cases where you know the resulting size of the slice it should be allocated accordingly to avoid growing steps:
```suggestion
	evmcBlobHashes := make([]evmc.Hash, 0, len(params.BlobHashes))
```